src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -156,7 +156,7 @@ private FeatureDefinition GetDotnetSchemaFeatureDefinition(string featureName)
             IEnumerable<IConfigurationSection> dotnetFeatureDefinitionSections = GetDotnetFeatureDefinitionSections();
 
             IConfigurationSection configuration = dotnetFeatureDefinitionSections
-                .FirstOrDefault(section =>
+                .LastOrDefault(section =>
```

Comment:
No change should be necessary for .NET featre definition schema.

Reply:
Reverted back to FirstOrDefault